### PR TITLE
chore(sdk): upgrade the widgets with latest SDK version

### PR DESCRIPTION
**Pull request by Neeraj-swarnkar**
**Description of changes made (Required)**
As part of SPARK-426299, Upgrade the widgets with latest SDK, components

**Associated JIRA (Required)**

JIRA
[SPARK-426299](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-426299)

**Relevant UI (Optional)**

https://app.vidcast.io/share/3bb6a78a-a309-44c4-a3bd-179e1f61f3f3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -96,7 +96,7 @@
     "sass-loader": "^8.0.2",
     "semantic-release": "^18.0.0",
     "style-loader": "^1.1.3",
-    "webex": "^1.149.2",
+    "webex": "^1.161.0",
```
Comment (sreenara) [inline package.json:99]: Why are we moving to 1.161? Why not 2.54.0. There are no breaking changes between 1.x.x and 2.x.x.

**Review by sreenara — CHANGES_REQUESTED** [review]
Just one question regarding the webex dependency

File: package.json (hunk 1 above)
Comment (arun3528) [reply package.json:99]: +1

Comment (Neeraj-swarnkar) [reply package.json]: @arun3528, @sreenara, I was going with minor updates and testing,

Comment (Neeraj-swarnkar) [reply package.json]: @arun3528, @sreenara, I have amended it with the latest version available, I will test the flows and update.

**Review by sreenara — APPROVED** [review]
LGTM

**Pull request merged**